lib/config/default.js

```diff
@@ -1706,7 +1706,28 @@ defaultConfig.definition = () => ({
      * application entity that is instrumented by the New Relic agent.
      */
     metrics: {
-      enabled: { default: false, formatter: boolean }
+      enabled: { default: false, formatter: boolean },
+
+      /**
+       * `exportInterval` defines the number of milliseconds between each
+       * attempt to ship metrics to New Relic. This value must be equal to
+       * or greater than the value of `exportTimeout`.
+       */
+      exportInterval: {
```

Comment:
so this default is the same as the agent metrics. it's worth noting we don't allow customers to configure the export interval(harvest interval)

Reply:
I assume you did this so you don't have to wait in the tests?